style_editor.html

```diff
@@ -2636,10 +2651,11 @@
   } else {
     sortString = "A:"+hsl[0];
   }
-  var color = "rgb("+r+","+g+","+b+")";
-  if (r == 0 && g == 0 && b == 0) color = "white";
-  return "<input srt='"+sortString+"' type=button style='border: 1px solid black;padding:8px;color:black;background:"+color+"' class=btn onclick='SetTo(\""+name+"\")' value='"+enc(name)+"'>\n";
+  var text = (name === "Black" || name === "Blue") ? "white" : "black";
+  var color = (name === "Black") ? "black" : "rgb("+r+","+g+","+b+")";
```

Comment:
I don't understand why we need a special case for black here.

Reply:
changed.